File: csharp/GetMunicipalityById.cs
using System;
using System.Diagnostics;
using Intrinio.SDK.Api;
using Intrinio.SDK.Client;
using Intrinio.SDK.Model;

namespace Example
{
    public class GetAllMunicipalityByIdExample
    {
        static void Main()
        {
            Configuration.Default.AddApiKey("api_key", "YOUR_API_KEY");

            var municipalityApi = new MunicipalityApi();
            var identifier = "geV70X";

            try
            {
                Municipality municipality = municipalityApi.GetMunicipalityById(identifier);

                Console.WriteLine(municipality.GovernmentName);
                Console.WriteLine("--------------------------------------");
                Console.WriteLine("ID:                    " + municipality.Id);
                Console.WriteLine("Census ID:             " + municipality.CensusId);
                Console.WriteLine("Government type:       " + municipality.GovernmentType);
                Console.WriteLine("Area type:             " + municipality.AreaType);
                Console.WriteLine("Area name:             " + municipality.AreaName);
                Console.WriteLine("Web site:              " + municipality.WebSite);
                Console.WriteLine("Population:            " + municipality.Population);
                Console.WriteLine("Population as of year: " + municipality.PopulationAsOfYear);
                Console.WriteLine("Enrollment:            " + municipality.Enrollment);
                Console.WriteLine("Enrollment as of year: " + municipality.EnrollmentAsOfYear);
                Console.WriteLine("Latitude:              " + municipality.Latitude);
                Console.WriteLine("Longitude:             " + municipality.Longitude);
                Console.WriteLine("Location:              " + municipality.Address1);
                Console.WriteLine("                       " + municipality.Address2);
                Console.WriteLine("                       " + municipality.City + ", " + municipality.State + " " + municipality.Zip);

            }
            catch (Exception e)
            {
                Debug.Print("Exception when calling MunicipalityApi.GetMunicipalityById: " + e.Message);
            }
        }
    }
}

File: python/get_municipality_by_id.py
from __future__ import print_function
import time
import intrinio_sdk
from intrinio_sdk.rest import ApiException
from pprint import pprint
 
intrinio_sdk.ApiClient().configuration.api_key['api_key'] = 'YOUR_API_KEY'
 
municipality_api = intrinio_sdk.MunicipalityApi()
 
identifier = 'geV70X' # str | An Intrinio ID of a Municipality
 
try:
    municipality = municipality_api.get_municipality_by_id(identifier)
    print(municipality.government_name)
    print("-------------------------------------------------")
    print("ID:                    %s" % (municipality.id))
    print("Census ID:             %s" % (municipality.census_id))
    print("Government type:       %s" % (municipality.government_type))
    print("Area type:             %s" % (municipality.area_type))
    print("Area name:             %s" % (municipality.area_name))
    print("Web site:              %s" % (municipality.web_site))
    print("Population:            %s" % (municipality.population))
    print("Population as of year: %s" % (municipality.population_as_of_year))
    print("Enrollment:            %s" % (municipality.enrollment))
    print("Enrollment as of year: %s" % (municipality.enrollment_as_of_year))
    print("Latitude:              %s" % (municipality.latitude)) 
    print("Longitude:             %s" % (municipality.longitude)) 
    print("Location:              %s" % (municipality.address1))
    if municipality.address2 is not None:
        print("                       %s" % (municipality.address2))
    print("                       %s, %s, %s" % (municipality.city, municipality.state, municipality.zip))
except ApiException as e:
    print("Exception when calling MunicipalityApi->get_municipality_by_id: %s\n" % e)
